data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,338 @@
+# Copyright (c) 2023 by RisingLeaf (https://github.com/RisingLeaf)
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Incipias: Guardian of Incipias Space"
+	invisible
+	landing
+	repeat
+	to offer
+		not "Incipias: Guardian of Incipias Space: active"
+	to complete
+		never
+	npc save
+		government "Drak (Incipias)"
+		personality heroic vindictive frugal forbearing
+		system Il'le
+		ship "Archon (Hyperdrive Only)" "Guardian of Spring"
+
+mission "Incipias: Warning of the Guardian"
```

Comment:
This feels too straightforward. Also, it's wildly different from how Archons are known to behave.

Reply:
Agreed. The Drak are not this bombastic and aggressive. I'd tone it down a little and keep it more in line with the Conversation on Sayaiban mission.

Reply:
updated the drak convo, to make it more calm and the style a bit more poetic?
please have a look